modules/test/playwright/tests/portal-workflow-task-web/workload.spec.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {notificationPagesTest} from '../../fixtures/notificationPagesTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import clearUserNotifications from '../../utils/clearUserNotifications';
+import retryGetWorkflowTasksBySubmittingUser from '../../utils/retryGetWorkflowTasksBySubmittingUser';
+import {blogsPagesTest} from '../blogs-web/fixtures/blogsPagesTest';
+
+export const test = mergeTests(
+	applicationsMenuPageTest,
+	apiHelpersTest,
+	blogsPagesTest,
+	loginTest(),
+	notificationPagesTest,
+	workflowPagesTest
+);
+
+interface CreatedEntities {
+	blogPosts?: TBlogPost[];
+	users?: TUserAccount[];
+}
+
+const createdEntities: CreatedEntities = {};
+
+test.afterEach(async ({apiHelpers, configurationTabPage, page}) => {
+	await configurationTabPage.goTo();
+
+	await configurationTabPage.unassignWorkflowFromAssetType('Blogs Entry');
+
+	if (createdEntities.users?.length) {
+		for (const user of createdEntities.users) {
+			await apiHelpers.headlessAdminUser.deleteUserAccount(
+				Number(user.id)
```

Comment:
Hello @cgmonte, in most of the scenarios I had the opportunity to work on, the `id` is stored as a number. Could you check if there is an error in the `TUserAccount` that was declaring the id as a string? or whether this would be the correct primitive type for this case.

Reply:
As we talked `TUserAccount` was defined by another team and is heavily used in other tests. Modifying from `string` to `number` would require changing too many files.